index.html

```diff
@@ -53,6 +53,120 @@ <h1>I’m Seko<br> Glad to <br>see you!</h1>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="techs">
+                        <li class="btn btn-secondary">html</li>
+                        <li class="btn btn-secondary">css</li>
+                        <li class="btn btn-secondary">javascript</li>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+            <div class="work-container">
+                <div class="work-image">
+                    <img src="/img/art printing.png" width="278px" height="220px" alt="">
+                </div>
+                <div class="work-content">
+                    <h2>Professional Art</h2>
+                    <ul class="work">
+                        <li class="work-item works-title">Art</li>
+                        <li class="work-item works-role"> <img src="/img/Counter.png" alt=""> Back End Dev</li>
+                        <li class="work-item works-year"> <img src="/img/Counter.png" alt=""> 2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="techs">
+                        <li class="btn btn-secondary">html</li>
+                        <li class="btn btn-secondary">css</li>
+                        <li class="btn btn-secondary">javascript</li>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+
+        <section class="about-section">
+            <div class="about-container">
+                <div class="desc">
+                    <h2 class="about-heading">About<br> Myself</h2>
+                    <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#" class="connect">Let's Connect</a>
+                    <div class="social-icon">
+                        <ul>
+                            <li><a href="#"><img src="/img/twiter icon.svg" alt="Twitter icon"></a></li>
+                            <li><a href="#"><img src="/img/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
+                            <li><a href="#"><img src="/img/medium icon.svg" alt="medium icon"></a></li>
+                            <li><a href="#"><img src="/img/github icon.svg" alt="github icon"></a></li>
+                            <li><a href="#"><img src="/img/hi icon.svg" alt="Hi icon"></a></li>
+                        </ul>
+                    </div>
+                    <button class="btn btn-primary">Get My Resume</button>
```

Comment:
- [ ] the button should follow the button styling: Active, Hover and Pressed kindly fix that like mentioned in Figma design 
![image](https://user-images.githubusercontent.com/89282221/191740811-4393f8c4-51ba-45ae-ab3f-f3a7c845c8b6.png)